### PR TITLE
Ensure all elements have an id property for testing purposes

### PR DESCRIPTION
**Pull request by ianatesfa**

### CODE REVIEW

**Comment by lazcool** [comment]
We only really need ids on elements that the automated tests will need access to, so e.g. we don't need id's for the layout divs.
Also hard-coding id's in foreach sections and in repeated partials will mean that the document will end up with duplicate ids.

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Shared/Error.cshtml [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
 }
 
-<h3>Development Mode</h3>
-<p>
+<h3 id="error-h3">Development Mode</h3>
+<p id="error-para1-dev">
     Swapping to <strong>Development</strong> environment will display more detailed information about the error that occurred.
 </p>
-<p>
+<p id="error-para1-dev">
```
Comment (lazcool) [inline src/SFA.DAS.FindEmploymentSchemes.Web/Views/Shared/Error.cshtml:20]: This is a dupe.

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml [hunk 2]
```diff
@@ -20,29 +20,31 @@
...
-	<div class="govuk-grid-column-one-third">
-		<h3 class="govuk-heading-m">Other schemes</h3>
-        <ul class="govuk-list">
+	<div id="scheme-details-inner-one-third" class="govuk-grid-column-one-third">
+		<h3 id="scheme-details-inner-h3" class="govuk-heading-m">Other schemes</h3>
+        <ul id="scheme-details-inner-ul" class="govuk-list">
             @foreach (var scheme in Model.OtherSchemes)
             {
-                <li><a asp-route="schemes" asp-route-schemeUrl="@scheme.Url" class="govuk-link">@scheme.Name</a></li>
+                <li id="scheme-details-inner-li">
```
Comment (lazcool) [inline src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml:45]: These will dupe.

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Shared/Error.cshtml (hunk 1 above)
Comment (ianatesfa) [reply src/SFA.DAS.FindEmploymentSchemes.Web/Views/Shared/Error.cshtml]: Ah, copy/paste mistake, fixed now

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml (hunk 2 above)
Comment (ianatesfa) [reply src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml]: Missed this loop, fixed in new commit

**Review by lazcool — APPROVED** [review]

**Pull request merged**